### PR TITLE
es: Create writing_guidelines/what_we_write

### PR DESCRIPTION
**Pull request by Graywolf9**
### Description

Create writing_guidelines/what_we_write

### Related issues and pull requests

Relates to https://github.com/mdn/translated-content/issues/14993

### CODE REVIEW

File: files/es/mdn/writing_guidelines/what_we_write/index.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+title: Lo que escribimos
+slug: MDN/Writing_guidelines/What_we_write
+l10n:
+  sourceCommit: 5cc673ab34acfb189832b22f85a44c6527e4a5ea
+---
+
+{{MDNSidebar}}
+
+MDN Web Docs contiene documentation _neutral al navegador_  que permite a los desarrolladores web escribir codigo _agnostico al navegador_. En este artículo, encontrarás información sobre si un tema o tipo de contenido determinado debe incluirse en MDN Web Docs.
```
Comment (JuanVqz) [inline files/es/mdn/writing_guidelines/what_we_write/index.md:10]:
```suggestion
MDN Web Docs contiene documentación _neutral al navegador_ que permite a los desarrolladores web escribir codigo _agnostico al navegador_. En este artículo, encontrarás información sobre si un tema o tipo de contenido determinado debe incluirse en MDN Web Docs.
```

Comment (Graywolf9) [reply files/es/mdn/writing_guidelines/what_we_write/index.md:10]: Y según yo releí XD muchas gracias!

File: files/es/mdn/writing_guidelines/what_we_write/index.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
+---
+title: Lo que escribimos
+slug: MDN/Writing_guidelines/What_we_write
+l10n:
+  sourceCommit: 5cc673ab34acfb189832b22f85a44c6527e4a5ea
+---
+
+{{MDNSidebar}}
+
+MDN Web Docs contiene documentation _neutral al navegador_ que permite a los desarrolladores web escribir codigo _agnostico al navegador_. En este artículo, encontrarás información sobre si un tema o tipo de contenido determinado debe incluirse en MDN Web Docs.
```
Comment (Graywolf9) [inline files/es/mdn/writing_guidelines/what_we_write/index.md:10]:
```suggestion
MDN Web Docs contiene documentación _neutral al navegador_ que permite a los desarrolladores web escribir código _agnostico al navegador_. En este artículo, encontrarás información sobre si un tema o tipo de contenido determinado debe incluirse en MDN Web Docs.
```

**Review by JuanVqz — APPROVED** [review]
@Graywolf9 Thanks!

**Pull request merged**